src/pages/DogEdit.js

```diff
@@ -1,9 +1,103 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useParams} from 'react-router-dom'
+import { Form, Input, Button, FormGroup, Label} from 'reactstrap'
+
+
+
+const DogEdit = (props) => {
+  const { id } = useParams()
+  const selectedDog = props.dogs?.find(dog => dog.id === +id)
+  const [newDog, setNewDog] = useState({
+    name: "",
+    age: "",
+    description: "",
+    image: "",
```

Comment:
Since you are editing the dog in this form, would it be helpful to display the values currently in the database of the dog you are editing so you know what to change to?